hooks/finding-post-processing/.helmignore

```diff
@@ -32,4 +32,5 @@ src/*
 config/*
 Dockerfile
 .dockerignore
-
+finding-post-processing.tar
```

Comment:
```suggestion
*.tar
```
I would recommend to exclude all kinds of tar files in all `.helmignore` files instead of an specific exclude